### PR TITLE
Lacework integration

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?
Adds Lacework integration

### Motivation
PM request

### Preview link

* https://docs-staging.datadoghq.com/gus/lacework/integrations/
* https://docs-staging.datadoghq.com/gus/lacework/integrations/lacework/

### CODE REVIEW

File: content/en/integrations/lacework.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+doc_link: /integrations/lacework/
+has_logo: true
+integration_title: lacework
+is_public: true
+public_title: Datadog-Lacework Integration
+---
+
+## Overview
+
+Use the Datadog-Lacework integration in order to forward your Lacework logs to Datadog.
```
Comment (ruthnaebeck) [inline content/en/integrations/lacework.md]:
```suggestion
Use the Datadog Lacework integration to forward your Lacework logs to Datadog.
```

File: content/en/integrations/lacework.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+short_description: "Forward your Lacework logs to Datadog."
+categories:
+- log collection
+- security
+doc_link: /integrations/lacework/
+has_logo: true
+integration_title: lacework
+is_public: true
+public_title: Datadog-Lacework Integration
+---
```
Comment (ruthnaebeck) [inline content/en/integrations/lacework.md]: Check https://docs-staging.datadoghq.com/gus/lacework/integrations/lacework/

The title is showing with a lowercase `l`. Not sure which one changes this.

File: content/en/integrations/lacework.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+## Overview
+
+Use the Datadog-Lacework integration in order to forward your Lacework logs to Datadog.
+
+## Setup
+
+This is a *push integration* which means that Lacework pushes logs/events to Datadog. All configuration needed happens on Lacework's dashboard. Datadog automatically enables the right logs processing pipeline when it detects Lacework logs.
```
Comment (ruthnaebeck) [inline content/en/integrations/lacework.md]:
```suggestion
Lacework pushes logs and events to Datadog, so all configuration is done on Lacework's dashboard. Datadog automatically enables the right logs processing pipeline when it detects Lacework logs.
```

File: content/en/integrations/lacework.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+### Configuration
+
+1. Go to *Settings* and select *Integrations*
```
Comment (ruthnaebeck) [inline content/en/integrations/lacework.md]:
```suggestion
1. In Lacework, go to *Settings* and select *Integrations*.
```

File: content/en/integrations/lacework.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+2. In the *Outgoing* section (on the left panel) select Datadog.
+3. Fill in the following details:
+    * **Name** : Enter a name for the integration like `Datadog-Lacework`
```
Comment (ruthnaebeck) [inline content/en/integrations/lacework.md]:
```suggestion
    * **Name** : Enter a name for the integration like `Datadog-Lacework`.
```

File: content/en/integrations/lacework.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+    * **Datadog Type**: Select the type of logs sent to Datadog:
+
+        | Datadog Type     | Description                                           |
+        |------------------|-------------------------------------------------------|
+        | `Logs Details`   | Sends Lacework detailed logs to Datadog logs platform. |
```
Comment (ruthnaebeck) [inline content/en/integrations/lacework.md]:
```suggestion
        | `Logs Details`   | Sends Lacework detailed logs to the Datadog logs platform. |
```

File: content/en/integrations/lacework.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+        | `Logs Summary`   | Sends Lacework summary to Datadog logs platform.       |
```
Comment (ruthnaebeck) [inline content/en/integrations/lacework.md]:
```suggestion
        | `Logs Summary`   | Sends a Lacework summary to the Datadog logs platform.       |
```

File: content/en/integrations/lacework.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+        | `Events Summary` | Sends Lacework summary to Datadog Events platform.     |
```
Comment (ruthnaebeck) [inline content/en/integrations/lacework.md]:
```suggestion
        | `Events Summary` | Sends a Lacework summary to the Datadog Events platform.     |
```

File: content/en/integrations/lacework.md [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+
+    * **Datadog Site**:
+        * Select `com` if you use Datadog US region.
+        * Select `eu` if you use Datadog EU region.
+    * **API KEY**: Enter your Datadog API key.
```
Comment (ruthnaebeck) [inline content/en/integrations/lacework.md]: Add a link to https://app.datadoghq.com/account/settings#api

File: content/en/integrations/lacework.md [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+## Setup
+
+Lacework pushes logs and events to Datadog, so all configuration is done on Lacework's dashboard. Datadog automatically enables the right logs processing pipeline when it detects Lacework logs.
+
+### Configuration
+
+1. In Lacework, go to *Settings* and select *Integrations*.
+2. In the *Outgoing* section (on the left panel) select Datadog.
+3. Fill in the following details:
+    * **Name** : Enter a name for the integration like `Datadog-Lacework`.
```
Comment (cswatt) [inline content/en/integrations/lacework.md]:
```suggestion
    * **Name**: Enter a name for the integration. For example, `Datadog-Lacework`.
```

File: content/en/integrations/lacework.md [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+    * **Datadog Type**: Select the type of logs sent to Datadog:
+
+        | Datadog Type       | Description                                                |
+        | ------------------ | -------------------------------------------------------    |
+        | `Logs Details`     | Sends Lacework detailed logs to the Datadog logs platform. |
+        | `Logs Summary`     | Sends a Lacework summary to the Datadog logs platform.     |
+        | `Events Summary`   | Sends a Lacework summary to the Datadog Events platform.   |
+
+    * **Datadog Site**:
+        * Select `com` if you use Datadog US region.
```
Comment (cswatt) [inline content/en/integrations/lacework.md]:
```suggestion
        * Select `com` if you use the Datadog US region.
```

File: content/en/integrations/lacework.md [hunk 12]
```diff
@@ -0,0 +1,50 @@
...
+        * Select `eu` if you use Datadog EU region.
```
Comment (cswatt) [inline content/en/integrations/lacework.md]:
```suggestion
        * Select `eu` if you use the Datadog EU region.
```

**Pull request merged**